app/models/test.rb

```diff
@@ -0,0 +1,5 @@
+class Test < ApplicationRecord
```

Comment:
Hey @tuannguyensn2001, could you provide more context to the relationship between Test, TestContent, TestMultipleChoice, and TestMultipleChoiceAnswer?

Reply:
ok
one test have many type, for example, multiple choice, essay, coding,... and my idea is each type is a table in database, and between them and test will have polymorphic relationship, it's test content, in test content it has test_id, type of test, and its id
for example, test is multiple choice, so that test_id is 1, testable_type is TestMultipleChoice, and testable_id is 1, it's id in table TestMultipleChoice

Reply:
It's a bit difficult to follow along, can you update the PR description to include the db diagram (https://dbdiagram.io/home) of those tables?